doc/ref_cert/RC1/chapters/chapter02.md

```diff
@@ -72,8 +44,7 @@ Conformance methodologies to be implemented, from a process perspective include:
 * Verification decisions will be based on data. Test harness is compatible, or conforms to testing against standard interfaces and services.
 * Leverage test harnesses from existing open source projects where practical, and applicable.
 
-<a name="2.3"></a>
-## 2.3 Conformance Strategy & Vehicle
+## Conformance Strategy & Vehicle
 In order to begin the Conformance process, NFVI needs to be validated and expected to be in a required state. This state would be determined by running tests as described in RI. Once the target NFVI passes these tests, it would become a candidate for NFVI Conformance. If NFVI fails the tests, it will not be move to next workflow for Conformance. NFVI+VNF conformance consist of a three part process for Compliance, Validation, and Performance. Adherence to Security standards are equally important and addressed in [Chapter 7 of Reference Model](../../../ref_model/chapters/chapter07.md).
```

Comment:
```suggestion
In order to begin the Conformance process, the Cloud Infrastructure needs to be validated and expected to be in a required state. This state would be determined by running tests as described in RI. Once the target Cloud Infrastructure passes these tests, it would become a candidate for the Cloud Infrastructure Conformance. If the Cloud Infrastructure fails the tests, it will not be moved to the next workflow for Conformance. The Cloud Infrastructure+VNF conformance consist of a three part process for Compliance, Validation, and Performance. Adherence to Security standards are equally important and addressed in [Chapter 7 of Reference Model](../../../ref_model/chapters/chapter07.md).
```